ansible/bench.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Verify Ansible versions
+  hosts: all
+  tags: always
+  become: false
+  run_once: true
+  gather_facts: false
+  tasks:
+    - local_action: command ./roles.py --check
+      changed_when: false
+
+- name: Configure Short Benchmarking Eth1 node
+  become: true
+  vars_files: group_vars/nimbus-bench.eth1.yml
+  hosts:
+    - bench-01.he-eu-hel1.nimbus.eth1
+  roles:
+    - { role: infra-role-open-ports,        tags: open-ports  }
+    - { role: infra-role-nimbus-bench-eth1, tags: nimbus-bench-short,  vars: { nimbus_eth1_benchmark_type: 'short'} }
+
+- name: Configure Long Benchmarking Eth1 node
+  become: true
+  vars_files: group_vars/nimbus-bench.eth1.yml
+  hosts:
+      - bench-02.he-eu-hel1.nimbus.eth1
+  roles:
+    - { role: infra-role-open-ports,        tags: open-ports  }
+    - { role: infra-role-nimbus-bench-eth1, tags: nimbus-bench-long,  vars: { nimbus_eth1_benchmark_type: 'long'} }
```

Comment:
would this var `nimbus_eth1_benchmark_type ` be better of in its own host var instead of this place?

Reply:
I think this is quite clear, it's good.